lib/graphql/schema/member/has_arguments.rb

```diff
@@ -58,6 +58,22 @@ def arguments
           end
         end
 
+        # @return [GraphQL::Schema::Argument, nil] Argument defined on this thing, fetched by name.
+        def get_argument(argument_name)
+          a = own_arguments[argument_name]
+
+          if a || !self.is_a?(Class)
```

Comment:
Turns out this method wasn't entirely correct since `HasArguments` needs to support being both `extend`ed and `include`ed. Using `ancestors` unconditionally breaks this implicit contract but didn't cause any issues until this PR so I had to support the instance case for fields.